### PR TITLE
[visq] Fix typos

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit fixes some typos in visq.

I'm trying to update my draft on visq_runner for one-vscode.

ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: compiler/visq/visqlib/QErrorComputer.py [hunk 1]
```diff
@@ -211,10 +211,15 @@ def run(self):
 class SRMSEComputer(QErrorComputer):
     def __init__(self, fp32_dir, fq_dir):
         super().__init__(fp32_dir, fq_dir)
-        self.scale_file = Path(fq_dir) / 'scales.txt'
+        if fq_dir != None:
```
Comment (jinevening) [inline compiler/visq/visqlib/QErrorComputer.py:214]: Is there a case when `fq_dir` is None?

Comment (stamalakhov) [reply compiler/visq/visqlib/QErrorComputer.py:214]: > Is there a case when `fq_dir` is None?

Yes.
See https://github.com/Samsung/ONE/blob/16dcb2f481ca650c6a4a49817d751ad84f3e039f/compiler/visq/visq#L364.

Comment (jinevening) [reply compiler/visq/visqlib/QErrorComputer.py:214]: Ah.. right. I think we have to change the constructor signature. Thanks!

**Review by jinevening — APPROVED** [review]
LGTM Thank you

**Pull request merged**